presto-product-tests/pom.xml

```diff
@@ -179,6 +225,14 @@
         </resources>
 
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>8</source>
+                    <target>8</target>
+                </configuration>
+            </plugin>
```

Comment:
Is this section necessary? Of so, we should probably have source and target set to `project.build.targetJdk`

ref: https://github.com/prestodb/airbase/blob/bfe642e8eeed477ba93364be2cd5e102ad62654a/airbase/pom.xml#L388-L389